scripts/create_additive_authoritative_structures.sh

```diff
@@ -150,8 +150,7 @@ function main {
   then
     build_authoritative
   fi
-
-  jq -n --arg output "$OUTPUT" '{data:$output}'
+  echo "$OUTPUT"| sed -e 's/.*/{\n  "data": "&/; s/$/"\n}/'
```

Comment:
Can you please explain this solution?

Reply:
@aaron-lane Oh, it very simple, in fact, we get the ready data in the `OUTPUT` variable in terms of further processing, so we need only wrap it to appropriate format. The previous solution is correct, but redundant in terms of both poor performance and inability to expand and collapse a large data array. In the future, it makes sense to get rid of this script and use new terraform functions to generate data

Reply:
I think there may be an issue with this change...

I'm trying out the changes from https://github.com/terraform-google-modules/terraform-google-iam/pull/44 which incorporate this change and was getting the following error: 
```
Error: command "bash" produced invalid JSON: invalid character 'n' looking for beginning of object key string

  on .terraform/modules/project_iam_binding/main.tf line 83, in data "external" "additive_bindings_temp_struct":
  83: data "external" "additive_bindings_temp_struct" {
```

When I swapped this line back to the old version in the downloaded module it worked:
```
jq -n --arg output "$OUTPUT" '{data:$output}'
```

The following will reproduce it...
```
module "project_iam_binding" {
  source  = "git::https://github.com/terraform-google-modules/terraform-google-iam.git"
  projects = ["my-project"]

  bindings = {
    "roles/storage.admin" = [
      "user:foo@something.com",
    ]
  }
}
```
Let me know if there are any other details I can provide.

Reply:
@jason-huling please open a new issue with this information so that we can more effectively address this.

Reply:
No problem @aaron-lane: https://github.com/terraform-google-modules/terraform-google-iam/issues/45